daemon/container_operations.go

```diff
@@ -0,0 +1,9 @@
+package daemon
+
+import "errors"
+
+var (
+	// ErrRootFSReadOnly is returned when a container
+	// rootfs is marked readonly.
+	ErrRootFSReadOnly = errors.New("container rootfs is marked read-only")
```

Comment:
agree, the `container` package is even imported where it's used in archive.go. Perhaps even push the error definition into the error package. What say you @duglin ?